client/src/parsec/terms_of_service.ts

```diff
@@ -0,0 +1,40 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { needsMocks } from '@/parsec/environment';
+import { getParsecHandle } from '@/parsec/routing';
+import { ClientAcceptTosError, ClientGetTosError, Result, Tos } from '@/parsec/types';
+import { libparsec } from '@/plugins/libparsec';
+import { DateTime } from 'luxon';
+
+export async function acceptTOS(updatedOn: DateTime): Promise<Result<null, ClientAcceptTosError>> {
+  const handle = getParsecHandle();
+
+  if (handle !== null && !needsMocks()) {
+    return await libparsec.clientAcceptTos(handle, updatedOn.toSeconds() as any as DateTime);
+  } else {
+    return { ok: true, value: null };
+  }
+}
+
+export async function getTOS(): Promise<Result<Tos, ClientGetTosError>> {
+  const handle = getParsecHandle();
+
+  if (handle !== null && !needsMocks()) {
+    const result = await libparsec.clientGetTos(handle);
+    if (result.ok) {
+      result.value.updatedOn = DateTime.fromSeconds(result.value.updatedOn as any as number);
+    }
+    return result;
+  } else {
+    return {
+      ok: true,
+      value: {
+        perLocaleUrls: new Map<string, string>([
+          ['en-US', 'https://www.youtube.com/watch?v=dQw4w9WgXcQ'],
+          ['fr-FR', 'https://www.youtube.com/watch?v=hEmODTcKJmE'],
```

Comment:
If there is even the slightest chance that this ends-up being shown to the user (see above comment), I would not put these URLs in the source code. We can just put some invalid placeholder URLs.

Reply:
I like the thought of rickrolling the user in case an error occurs. But yeah, you're right, I'll replace those.

Reply:
Me too, and this is probably not a big issue with SaaS users. But I was thinking specially in the context of Parsec as a security solution for government agencies or in the context of product certifications.